biomage/experiment/pull.py

```diff
@@ -52,16 +73,16 @@ def update_config_if_needed(file, table_name, experiment_id):
         remote_cfg = remove_key(remote_cfg, "pipeline")
 
     # if the local config was not found or it's different from the remote => update
-    if not found or DeepDiff(local_cfg, remote_cfg):
-        save_cfg_file(remote_cfg, file)
-        Summary.add_changed_file(file)
+    if not found or local_cfg != remote_cfg:
+        save_cfg_file(remote_cfg, filepath)
+        Summary.add_changed_file(filepath)
 
 
 def update_configs(experiment_id, origin):
     # config pairs like: (local file name, remote table name)
     configs = [
-        ("mock_experiment.json", "experiments"),
-        ("mock_samples.json", "samples"),
+        (EXPERIMENTS_FILE, EXPERIMENTS_TABLE),
+        (SAMPLES_FILE, SAMPLES_TABLE),
```

Comment:
that's much better than hardcoding them. Eventually, we can move them to config file (I know that Agi created one as part of his PR)

Reply:
yes, that would be optimal